releasenotes/notes/start-agent-on-configure-595265595536fd46.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Start the agent service on configure if it is not running
```

Comment:
```suggestion
    Start the Agent service when "configure" is clicked from the systray, if it is not running
```

Reply:
thanks for the suggestion! confirming it should be `Configure`